generate_admin_token.py

```diff
@@ -1,7 +1,10 @@
 #! /usr/bin/python
 import sys
+import socket
 import urllib, urllib2
 
+host_ip = socket.gethostbyname(socket.gethostname())
```

Comment:
I am confused about the changes in this file.  Is this file still functional?
For this line, host_ip is not used, so this line should be called.